tests/functional/object/mcg/test_noobaa_db_cleartext_postgres_password.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from ocs_ci.framework.testlib import tier2, BaseTest, bugzilla
+from ocs_ci.framework.pytest_customization.marks import red_squad, mcg
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources import pod
+
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("2274193")
+class TestNoobaaSecurity(BaseTest):
+    """
+    Test Noobaa Security
+    """
+
+    @mcg
+    @red_squad
+    @bugzilla("2274193")
```

Comment:
Remove the duplicate marker, you added the bugzilla already above

Reply:
https://github.com/red-hat-storage/ocs-ci/pull/9806#issuecomment-2119168967